spec/index.bs

```diff
@@ -1850,7 +1865,29 @@ interface GPUQueue {
 GPUQueue includes GPUObjectBase;
 </script>
 
- - {{GPUQueue/copyImageBitmapToTexture()}}:
+{{GPUQueue/writeBuffer()}} takes the `sourceData` contents starting from the offset `sourceOffset` and
```

Comment:
Instead of xrefs, these should be the primary definitions of writeBuffer and writeTexture, similar to [createBuffer](https://github.com/gpuweb/gpuweb/blob/ce4132a9d1eadab2d5668037c915b7c811f68198/spec/index.bs#L672)